### PR TITLE
Release 12/06: Update to Network Flow guided install

### PR DESCRIPTION
**Pull request by nr-dlaturner**
Added update to network flow guided install to reflect current install process.

Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* What problems does this PR solve?
* Add any context that will help us review your changes such as testing notes,
  links to related docs, screenshots, etc.
* If your issue relates to an existing GitHub issue, please link to it.

### CODE REVIEW

**Comment by nr-dlaturner** [comment]
Context for these PRs.  

https://github.com/newrelic/docs-website/pull/10582
https://github.com/newrelic/docs-website/pull/10584
https://github.com/newrelic/docs-website/pull/10585

These 3 PRs are submitted to update the documentation for the guided install of network monitoring components snmp, network flow and syslog.   It changes some of wording to better match what is actually in the add data steps.  It corrects drift between the features available through adding data and the documentation supporting this process.

File: src/content/docs/network-performance-monitoring/setup-performance-monitoring/network-flow-monitoring.mdx [hunk 1]
```diff
@@ -368,13 +368,13 @@ When planning your strategy for collecting network flows at scale, New Relic rec
 ## Set up network flow data monitoring in New Relic [#setup-network-flow-monitoring]
 
 1. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Add more data**.
-2. Scroll down until you see **Network monitoring** and click **Network Flows**.
-3. Follow the steps in New Relic.
+2. Scroll down until you see **Network** and click **Network Flows**.
+3. Follow the steps outlined in the New Relic UI.  The available installation methods are a Docker or Linux package manager.
```
Comment (nr-dlaturner) [inline src/content/docs/network-performance-monitoring/setup-performance-monitoring/network-flow-monitoring.mdx:372]: Can this be changed to - "The available installation methods are Docker or Linux package manager."  Removing the "a".

Comment (nr-dlaturner) [inline src/content/docs/network-performance-monitoring/setup-performance-monitoring/network-flow-monitoring.mdx:372]: Can this be changed to - "The available installation methods are Docker or Linux package manager."  Removing the "a".

**Review by homelessbirds — APPROVED** [review]

**Review by nr-dlaturner — COMMENTED** [review]
Looks good

**Comment by rhetoric101** [comment]
I am temporarily closing this because the build seems to be stuck.

**Comment by rhetoric101** [comment]
I merged in develop. I am temporarily closing to trigger a new build.

**Pull request merged**